### PR TITLE
Create new args and new function

### PR DESCRIPTION
**Pull request by BeepedWhenIShouldaBooped**
Adding functionality to read a CSV from a URL provided in the command line
binx.py: -gt switch added which accepts a quote-enclosed URL and the binary strings file as arguments
strings.py: added a new function (translate_binary_file_with_url) that will read the URL contents and place in a temp file before passing to (translate_binary_file_with_csv)
__init__.py: added the string for the argument handler

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .vscode/
 __pycache__
-*.pyc
\ No newline at end of file
+*.pyc
+
```
Comment (pistacuro) [inline .gitignore:4]: @BeepedWhenIShouldaBooped don't add tmp, bin, bytes files into gitignore. gitignore is mostly for things which are produced by your development work, like caches, editor config etc.

**Comment by pistacuro** [comment]
just one comment everything else looks good... +1

**Review by pistacuro — CHANGES_REQUESTED** [review]

**Comment by BeepedWhenIShouldaBooped** [comment]
I removed the changes to .gitignore
I commented the code to the new function

File: .gitignore (hunk 1 above)
Comment (BeepedWhenIShouldaBooped) [reply .gitignore:4]: I removed the changed (well, corrected the file and recommitted it)

**Pull request merged**